core/src/io/Dataparser.cpp

```diff
@@ -1059,32 +1070,31 @@ namespace IO
             myfile.Read_String( ovf_data_representation, "# Begin: Data" );
             std::istringstream repr( ovf_data_representation );
             repr >> ovf_data_representation;
-            if( ovf_data_representation == "Binary" ) 
+            if( ovf_data_representation == "binary" ) 

```

Comment:
I believe the above will not lowercase the entire line and will therefore not lowercase `ovf_data_representation`. It may be that this way MuMax files cannot be read in anymore, as they use "Data Binary" and "Data Text".

Reply:
MuMax would be read in without a problem since `Filter_File_Handler::GetLine()` will convert every line to the header to lower case. That case will work like that:
- The whole line is de-capitalized by `Filter_File_Handler::GetLine()`
- In that line we are searching for the string `"# Begin: Data"` which of course will be de-capitalized by `Read_String()` otherwise it would never match to the already de-capitalized line.
- There are 3 possible states for the variable `ovf_data_representation`. Those are `"text"`, `"binary 4"` or `"binary 8"`. In case of binary we want to discard the binary length.
- We initialize the `std::istringstream repr` with `ovf_data_representation` variable and we are keeping only the first word. ( `>>` would stop in `" " `).